server/tests/auth/test_github_auth_mode.py

```
@@ -0,0 +1,71 @@
+
+The crux is the deprecation split: NEW OAuth connections (login/CTA) are
+gated separately from honouring EXISTING OAuth tokens, so deprecating OAuth
+onboarding never orphans users who connected via OAuth before the App.
+"""
+
+import importlib
+
+from utils.auth import github_auth_mode as m
+
+
+def _set_mode(monkeypatch, value):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_set_mode`

Add return type annotation: `None`

(ANN202)